mobile-app/app/components/BottomSheetModal.web.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react'
+import { View } from 'react-native'
+import { ThemedProps } from './themed'
+import Modal from 'react-overlays/Modal'
+import { BottomSheetNavScreen } from './BottomSheetWithNav'
+
+type Props = ThemedProps & {
+  children: ReactElement
+  screenList: BottomSheetNavScreen[]
+  isModalDisplayed: boolean
+}
+
+export const BottomSheetModal = React.forwardRef((props: Props, ref: React.Ref<any>): JSX.Element => {
+  const {
+    children
+  } = props
+
+  return (
+    <Modal
+      container={ref as React.RefObject<any>}
+      show={props.isModalDisplayed}
+      renderBackdrop={() => <View style={{
+        position: 'absolute',
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        backgroundColor: 'black',
+        opacity: 0.3
+      }}
+                            />}
+      style={{
```

Comment:
Can't use `style={[]}` so opted-out of using tailwind, need to use `fixed` as it's the most precise-ish in terms of rendering position.